basic_module/account/src/types.rs

```diff
@@ -87,3 +82,21 @@ impl Action {
         unimplemented!()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test() {
```

Comment:
We cannot test anything with this test because it requires a help from a human vision. It's better to make an assertion.